frontend/amp/components/elements/RichLinkBlockComponent.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { css } from 'emotion';
+import { palette } from '@guardian/pasteup/palette';
+import { serif, sans } from '@guardian/pasteup/fonts';
+import { pillarPalette } from '@frontend/lib/pillars';
+
+export const RichLinkBlockComponent: React.SFC<{
+    element: RichLinkBlockElement;
+    pillar: Pillar;
+}> = ({ element, pillar }) => {
+    if (element.sponsorship) {
+        throw new Error('Sponsored rich links not supported');
+    }
+    return (
+        <aside className={style}>
+            <a className={linkStyle(pillar)} href={element.url}>
+                {element.text}
+            </a>
+        </aside>
+    );
+};
+
+const style = css`
```

Comment:
Very minor, but can we use a more descriptive/specific name than 'style'?

Reply:
Agreeing to both of these, just wanted to get them looking and working right. Will tidy up.